README.md

```diff
@@ -1 +1,23 @@
 # react-cli
+
+This repo is an exploration for making the tooling for React that provides the best developer experience.
+
+
+## Getting started should be dead simple
+
+Something like `npm install -g react-cli` then `react init cool-project`.
+
+## Gradual integration
+
+The initial setup should be super slim, but then you can add more to it. When you start to want to test your app, you can do `react add test`, if you want to use flow then `react add flow`, wanna add crash reporting then `react add sentry`...
```

Comment:
Just to clarify, I wasn't proposing that you use a 2 step process of 
```sh
npm install redux
react (link|install) redux
```
I think that `react install redux` should actually be smart enough to run `npm install` behind the scenes. The less steps the user need to do the more "friendly" it'll be.

`ember-cli` uses such an approach so that to install an add on all you need to run is `ember install addon`